packages/date-picker/src/FilterDateRangePicker/FilterDateRangePicker.tsx

```diff
@@ -90,6 +90,18 @@ export const FilterDateRangePicker = ({
       : null
   )
   const [isOpen, setIsOpen] = useState<boolean>(false)
+  const [startMonth, setStartMonth] = useState<Date>(
+    selectedRange?.from || defaultMonth || new Date()
```

Comment:
`selectedRange` get's referenced a lot, could it be placed in a variable at the very top and referenced all the way down?

Reply:
You mean `selectedRange?.from`? Probably 🤔